spec/Types.tex

```diff
@@ -249,6 +250,21 @@ \subsection{The String Type}
 sets, possibly including internationalization.
 \end{openissue}
```

Comment:
While there, let us update Strings above to say `comprised of Unicode characters in UTF-8 encoding`. Or perhaps simply `comprised of UTF-8 characters`.

Reply:
For the "different character sets"  openissue, I suggest removing it and instead saying something like "To represent another character set or encoding, the \chpl{bytes} type can be used."

Reply:
It is my understanding that it is still an open issue, technically? I have recently updated the docs to state that as well: https://chapel-lang.org/docs/master/usingchapel/chplenv.html#character-set

That being said, I can also add your suggestion to the existing open issue. `Currently, \chpl{bytes} can be used to represent another encoding`?

Reply:
Off-line, Brad suggested removing the "future support for different character sets" openissue because the possibility we will have it is very remote, at the moment.

Note that we still are planning to support different character sets and encodings in the near future. However, this is in a different context - when communicating strings to and from the user, ex. I/O. The internal representation of a `string` will remain UTF-8.

Reply:
on an offline discussion, we decided to drop the openissue as per Vass' initial suggestion